src/foolscap/test/common.py

```diff
@@ -20,6 +22,28 @@
 from twisted.python import failure
 from twisted.internet.main import CONNECTION_DONE
 
+def convert_byte(x):
+    """ Convert to bytes type if x is a text type, else return as such """
```

Comment:
Similar to comment about `map_bytes` because that's all that uses this

Reply:
Got it.